### DISCUSSION TITLE
cannot import module in vscode

### DISCUSSION THREAD

**Discussion by ivyy** [modular/modular#810] [Q&A] [answered]
I'm new to mojo, I setup a mojo project with all the mojo files in the same dir.  following the offical manual, I create a module file  mymodule.mojo :

```
struct MyPair:
    var first: Int
    var second: Int

    fn __init__(inout self, first: Int, second: Int):
        self.first = first
        self.second = second

    fn dump(self):
        print(self.first, self.second)
```

and try to import mymodule in the main.mojo file

```
from mymodule import MyPair
fn main():
    let mine = MyPair(4,2)
    mine.dump()
```

but when I tried to run the main.mojo, error occurs: 

```
~/demo$ mojo main.mojo
main.mojo:1:22: error: module 'mymodule' does not contain 'MyPair'
from mymodule import MyPair
```
do I miss any steps or configurations? please help.

**Comment by drunkwcodes** [comment, marked as answer]
You may make a directory like this:
```
.
├── my_package
│   ├── __init__.mojo
│   ├── mymodule.mojo
│   └── main.mojo
```

And use relative import in `main.mojo`:
`from .mymodule import MyPair`

However I think your approach should work.
It can be an issue.

**Reply by ivyy** [reply]
It solves my problems. thanks @drunkwcodes
